packages/api/src/webhooks/cannon.ts

```diff
@@ -222,6 +251,62 @@ export default class WebhookCannon {
     );
   }
 
+  async notifyFailedWebhook(
+    trigger: messages.WebhookTrigger,
+    params?: RequestInitWithTimeout,
+    err?: any
+  ) {
+    if (!trigger.user.emailValid) {
+      console.error(
+        `Webhook Cannon| User email is not valid, id: ${trigger.id}, streamId: ${trigger.stream?.id}`
+      );
+      return;
+    }
+    if (!(this.supportAddr && this.sendgridTemplateId && this.sendgridApiKey)) {
+      console.error(
+        `Webhook Cannon| Unable to send notification email to user, id: ${trigger.id}, streamId: ${trigger.stream?.id}`
+      );
+      console.error(
+        `Sending emails requires supportAddr, sendgridTemplateId, and sendgridApiKey`
+      );
+      return;
+    }
+
+    let signatureHeader = "";
+    if (params.headers[SIGNATURE_HEADER]) {
+      signatureHeader = `-H  "${SIGNATURE_HEADER}: ${params.headers["Livepeer-Signature"]}"`;
```

Comment:
is this used anywhere? also, the reference in assignment can be updated to the variable:

```ts
signatureHeader = `-H  "${SIGNATURE_HEADER}: ${params.headers[SIGNATURE_HEADER]}"`;
```

Reply:
Thanks for the catch!

Anyway, this is used in case the webhook is configured with a sharedSecret, so we could send a correct curl example via email to the user to test the call itself. 
However right now the part of the email where we provide the example curl is commented until I make a different template on sendGrid to support HTML, or it would come out as a non-readable blob of text